markdown/2017-10-02-welcome.md

```diff
@@ -0,0 +1,380 @@
+---
+title: Weekly Digest
+show_social: true
+show_agenda: true
+show_sponsorship: true
+...
+
+---
+
+Hello, and welcome to another year at the Department of Computing!
+<br>Regardless of whether you're a returning student or just started, this
+(albeit rather sizeable) email contains incredibly useful information, so keep
+on reading!
+
+---
+
+# What is DoCSoc?
+DoCSoc is the Department of Computing Society at Imperial College London!
+
+This year it is brought to you by:
+
+- **President**: Maurice Yap
+- **Vice President**: Thomas Bower
+- **Treasurer**: Levente Kurusa
+- **Secretary**: Nik Vangerow (that's me!)
+- **Academic Events Coordinator**: Rachel Mekhtieva Lee
+- **Industrial Events Coordinator**: Yoon Kim
+- **Social Events Coordinator**: Diana Avram
+- **Events Officers**: Alessandro Bonardi, Jack Pordi, Rohan Pritchard
+- **Marketing Officer**: Inara Ramji
+- **Webmaster**: Elliot Greenwood
+- **Game Dev Officer**: Uno Perwironegoro
+
+Throughout the year you will also get to know your departmental representatives
+(Dep Reps). They represent the student body (you) to the department.
+
+This year they are:
+
+- **Computing**: Martin Zlocha
+- **Joint Maths and Computing (JMC)**: Rachel Mekhtieva Lee
+- **Electronics and Information Engineering (EIE)**: Mwanakombo Hussein
+
+For details on how to get involved, just get in touch with us. We're always
+looking for volunteers to help us set up and tear down the events. If you would
+like to volunteer, please join our [DoCSoc: Operations Facebook
+group](https://www.facebook.com/groups/1509850436007222/).
+
+## What Do We Do?
+DoCSoc is run by students, for students. With the generous support of our
+sponsors, we run events such as theatre trips, talks, and our infamous bar
+nights (psst, all drinks are on DoCSoc)! Our membership base is mostly comprised
+of Computing, JMC, and EIE students, but membership can also be purchased on the
+[union
+website](https://www.imperialcollegeunion.org/activities/a-to-z/computing) for
+anyone not on a DoC course. Why not get your friends on other courses to become
+members too?
+
+To stay up-to-date on the events that DoCSoc runs, keep an eye out for the
+weekly digest (you're reading it right now) which is baked and then delivered
+fresh to your inbox every week! Additionally, liking our [Facebook
+page](https://www.facebook.com/ICDocSoc/) will help you stay up to date on
+everything going on with DoCSoc.
+
+## Our Sponsors
+Without the support of our generous sponsors, much of what we do and what makes
+DoCSoc so special would not be possible! Sponsors host many of our talks and
+will occasionally let you know of their great job opportunities.
+
+Which brings me on to why DoCSoc gets sponsorship: It's important to realise
+that our sponsors are looking to hire YOU! You get to build your career while we
+get the money to run awesome social events. To kick things off, we will be
```

Comment:
I probably wouldn't say "we get the money" - maybe something like "while we are given the tools to" or something. Just seems a little bit odd to start talking about the money straight away.